### PR TITLE
Json output

### PR DESCRIPTION
**Pull request by philnash**
Adds an `--output-format` (or `-o`) flag to the `deploy`, `list-templates`, `list` and `promote` commands. Commands that already supported this flag include `logs` and `env:list`. Commands that do not support an output format currently include `new` and `start`.

This supersedes #143.

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: packages/runtime-handler/src/dev-runtime/server.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { ServerlessFunctionSignature } from '@twilio-labs/serverless-runtime-types/types';
 import bodyParser from 'body-parser';
-import EventEmitter from 'events';
```
Comment (dkundel) [inline packages/runtime-handler/src/dev-runtime/server.ts:3]: What's the purpose of this change?

File: packages/twilio-run/src/commands/shared.ts [hunk 2]
```diff
@@ -11,3 +11,5 @@ export type ExternalCliOptions = {
     version: string;
   };
 };
+
+export type OutputFormat = string | undefined;
```
Comment (dkundel) [inline packages/twilio-run/src/commands/shared.ts:15]: should this be:
```suggestion
export type OutputFormat = 'json' | undefined;
```

File: packages/twilio-run/src/config/deploy.ts [hunk 3]
```diff
@@ -112,6 +114,7 @@ export async function getConfigFromFlags(
   }
 
   const { region, edge, runtime } = flags;
+  const outputFormat = flags.outputFormat || externalCliOptions?.outputFormat;
```
Comment (dkundel) [inline packages/twilio-run/src/config/deploy.ts:117]: Do we still need the concept of `externalCliOptions.outputFormat` if we support this as a flag?

**Review by dkundel — COMMENTED** [review]
Left a couple of comments. Looks good overall

File: packages/twilio-run/src/commands/shared.ts (hunk 2 above)
Comment (philnash) [reply packages/twilio-run/src/commands/shared.ts:15]: That clashes with the `externalCliOptions` version of `outputFormat` which has it set to `string | undefined`.
